crates/holochain_cascade/src/authority.rs

```diff
@@ -48,6 +50,23 @@ pub async fn handle_get_record(
     Ok(results)
 }
 
+/// Handler for get_by_op_type query to a Record authority
+// TODO: for sharding to work, this must be the authority of the op type
```

Comment:
Is this the place for this TODO note? All other functions in this file are also sharding-agnostic -- they already assume that you are the authority for the context of that call.

In the other functions here the actual check of whether you are an authority has been going in the level above: in the `CascadeImpl` -- maybe the todo should go there?